plugin/src/test/java/org/owasp/benchmarkutils/score/CweNumberTest.java

```diff
@@ -0,0 +1,74 @@
+package org.owasp.benchmarkutils.score;
```

Comment:
Do we have preambles/copyright notices for test files? I think we do. Can you add one here?

Reply:
Done